package.json

```diff
@@ -39,18 +40,16 @@
     "needle": "3.2.0",
     "openpgp": "5.9.0",
     "prettier": "2.8.8",
-    "request": "2.88.2",
```

Comment:
Are you sure about this one ? request is still used in gulpfile.js no ? @stanislavhh